contracts/lido_terra_hub/src/unbond.rs

```diff
@@ -131,6 +131,10 @@ pub fn execute_withdraw_unbonded(
     let unbonding_period = params.unbonding_period;
     let coin_denom = params.underlying_coin_denom;
 
+    if params.paused.unwrap_or(false) {
```

Comment:
I suggest moving the pause checking to the very beginning of the fn like it's done in the rest of the pausable fns. 
```rust
/// Handler for tracking slashing
pub fn execute_slashing(mut deps: DepsMut, env: Env) -> StdResult<Response> {
    let params: Parameters = PARAMETERS.load(deps.storage)?;
    if params.paused.unwrap_or(false) {
        return Err(StdError::generic_err("the contract is temporarily paused"));
    }

    ....
```
From my pov, it's better because of three reasons:
1. we stick with a kind of pattern that's used in the other functions
2. it's more readable if we check a pause in the beginning
3. if the contract is paused, the variables assignment is inefficient

Reply:
actually this is very true, we simply spend less gas if we check first

Reply:
thanks 👍 fixed